app/src/main/java/org/mozilla/fenix/compose/TabUrl.kt

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.sp
+import mozilla.components.browser.toolbar.MAX_URI_LENGTH
+import mozilla.components.lib.publicsuffixlist.PublicSuffixList
+import org.mozilla.fenix.ext.toShortUrl
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Default layout for a tab composable caption.
+ *
+ */
+@Composable
+fun TabUrl(
+    url: String,
+    publicSuffixList: PublicSuffixList,
+    modifier: Modifier = Modifier,
+    fontSize: TextUnit = 12.sp,
+    color: Color = FirefoxTheme.colors.textSecondary
+) {
+
+    fun getUrl(): String {
```

Comment:
Can we refactor this into support-ktx/utils if it's used in multiple places?

Reply:
This is already an extension function in Fenix :) [see](https://github.com/mozilla-mobile/fenix/blob/e6cae9b8007051df0a72931b3361654c4c018a22/app/src/main/java/org/mozilla/fenix/ext/String.kt#L43) 
I think uplifting it to AC could be a separated ticket as it's outside of the scope of this PR.

Reply:
I filed https://github.com/mozilla-mobile/fenix/issues/22770 for it